app/tests/unit/pages/projects.test.tsx

```diff
@@ -136,4 +136,16 @@ describe("The projects page", () => {
       screen.queryByLabelText("Filter by Project Managers")
     ).toBeInTheDocument();
   });
+
+  it("applys correct filter variables, to mutation", () => {
```

Comment:
this is not a mutation, we're just changing the query parameters
```suggestion
  it("renders the page with the status query variable from the status filter", () => {
```